internal/mode/static/nginx/config/servers.go

```diff
@@ -86,6 +91,18 @@ func executeServers(conf dataplane.Configuration) []executeResult {
 	return allResults
 }
 
+// getIPFamily returns whether the server should be configured for IPv4, IPv6, or both.
+func getIPFamily(baseHTTPConfig dataplane.BaseHTTPConfig) http.IPFamily {
+	switch ip := baseHTTPConfig.IPFamily; ip {
```

Comment:
```suggestion
	switch baseHTTPConfig.IPFamily {
```

You don't need to create the `ip` variable in the switch statement since you don't use it in any of the cases.